docs/getting_started/typescript/hello_world_in_typescript/index.md

```diff
@@ -88,142 +158,155 @@ Create the `package.json` file in the root of your project and add the following
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-deprecation": "^1.2.1",
-    "mocha": "8.x",
-    "nodemon": "^2.0.12",
-    "prettier": "^2.8.8",
-    "ts-node": "^10.8.1",
-    "typescript": "^4.4.2"
-  }
-}
 ```
 
-You should review a few parts of the `package.json`, and the first is the `scripts` section. These are the `npm` commands you'll use to build, lint, and start your application code.
-
 Next, examine the packages listed as dependencies. These are the packages that compose the Temporal TypeScript SDK, and each package maps to the four parts of a Temporal application: an Activity, Client, Worker, and Workflow. There is also [Nanoid](https://npmjs.com/package/nanoid), an `npm` package which you'll use to generate a unique identifier for your Workflow.
 
-Finally, look through the `devDependencies` section. These are the packages that let you set up a Node.js project with [Nodemon](https://www.npmjs.com/package/nodemon) (a tool that automatically restarts your application when it detects a change in your code), TypeScript, ESLint, and ts-node.
+```json
+"dependencies": {
+  "@temporalio/activity": "^1.11.5",
+  "@temporalio/client": "^1.11.5",
+  "@temporalio/worker": "^1.11.5",
+  "@temporalio/workflow": "^1.11.5",
+  "nanoid": "3.x"
+},
+```
 
-Save the file, then download the dependencies specified in the `package.json` file with the command:
+You'll use Node.js's build-in `fetch` library in your application. To use it with TypeScript, open the file `tsconfig.json` and locate the `"lib"` key under `"compilerOptions"`and add the `DOM` library to the array:
 
-```command
-npm install
+```json
+"compilerOptions": {
+ ...
+  "lib": ["es2020","DOM"],
+ ...
+}
 ```
 
-Downloading the dependencies can take a few minutes to complete. Once the download is done, you will have new a `package-lock.json` file and a `node_modules` directory.
+This ensures that `fetch` will be available as a global module.
 
-Your project workspace is configured, so you're ready to create your first Temporal Activity and Workflow. You'll start with the Workflow.
+With the project created, you'll create the application's core logic.
 
-## Create a Workflow
+## Write functions to call external services
 
-Workflows are where you configure and organize the execution of Activities. You define a Workflow by writing a _Workflow Definition_ using one of the Temporal SDKs. You can learn more in the [Develop Workflows](https://docs.temporal.io/dev-guide/typescript/foundations/#develop-workflows) section of the Temporal documentation.
+Your application will make two HTTP requests. The first request will return your current public IP, while the second request will use that IP to provide city, state, and country information.
 
-To begin your Workflow, create a new subdirectory called `src`:
+You'll use Temporal Activities to make these requests. You use Activities in your Temporal Applications to execute non-deterministic code or perform operations that may fail.
```

Comment:
I think it may be beneficial to explain WHY you use Activities to make these external requests, because a first-time reader would perhaps wonder why we don't do this in Workflows. 

Consider adding something like this:

"One of the key benefits of using Activities is their built-in fault tolerance. If an Activity fails, Temporal can automatically retry it until it succeeds or reaches a specified retry limit. This ensures that transient issues, like network glitches or temporary service outages, don't result in data loss or incomplete processes."

Reply:
This also gives a preview before line 212. Because if I read line 212 without an introduction to Retry Policies first, I would wonder what that meant.

Reply:
```suggestion
You'll use Temporal Activities to make these requests. You use Activities in your Temporal Applications to execute non-deterministic code or perform operations that may fail.

If an Activity fails, Temporal can automatically retry it until it succeeds or reaches a specified retry limit. This ensures that transient issues, like network glitches or temporary service outages, don't result in data loss or incomplete processes.
```